articles/quickstarts/qrng.md

```diff
@@ -22,9 +22,22 @@ A simple example of a quantum algorithm written in Q# is a quantum random number
 
 ### Q# operation code
 
-1. Replace the contents of the Operation.qs file with the following code:
-
- :::code language="qsharp" source="~/quantum/samples/getting-started/qrng/Qrng.qs" range="3-14":::
+1. Replace the contents of the Program.qs file with the following code:
+
+    ```qsharp
```

Comment:
As discussed offline, we should make sure this works with code in the samples repository rather than embedding directly into the article. See https://github.com/microsoft/Quantum/pull/338 for @bettinaheim's work towards doing so.

Reply:
Updated. Please take a look if it is okay.

Reply:
It looks like the source code that's getting pulled in is from master:
![image](https://user-images.githubusercontent.com/31516/80620605-0d3b2d00-89fb-11ea-80a9-7c79fb052e3e.png)

We'll want that for the final PR, of course, but it's hard to test rendering that way. As discussed offline, you can **temporarily** modify https://github.com/MicrosoftDocs/quantum-docs-pr/blob/build/0.11.2004.2825/.openpublishing.publish.config.json#L75 to point to the `feature/standalone` branch of the samples repo to get a test rendering.

Reply:
Ah, sorry, I had missed that the code here is copied-and-pasted. Please make sure to use something like this to embed code from the samples repository instead:

```
:::code language="qsharp" source="~/quantum/samples/getting-started/qrng/Qrng.qs" range="3-14":::
```